app/views/shared/_navigation.haml

```diff
@@ -13,4 +13,12 @@
       %label.govuk-label.navigation_heading
         = t('footer.for_schools')
       - unless ReadOnlyFeature.enabled?
-        %li{ class: active_link_class(new_identifications_path) }= link_to t('nav.sign_in'), new_identifications_path, class: 'govuk-header__link mobile-header-underline-text'
+        - if JobseekerAccountsFeature.enabled?
+          - if jobseeker_signed_in?
+            %li{ class: active_link_class(jobseekers_saved_jobs_path) }= link_to t('footer.your_account'), jobseekers_saved_jobs_path, class: 'govuk-header__link mobile-header-underline-text'
+            %li{ class: active_link_class(destroy_jobseeker_session_path) }= link_to t('nav.sign_out'), destroy_jobseeker_session_path, method: :delete, class: 'govuk-header__link mobile-header-underline-text'
+          - else
+            %li{ class: active_link_class(new_jobseeker_session_path) }= link_to t('buttons.sign_in'), new_jobseeker_session_path, class: 'govuk-header__link mobile-header-underline-text'
+            %li{ class: active_link_class(new_identifications_path) }= link_to t('nav.for_schools'), new_identifications_path, class: 'govuk-header__link mobile-header-underline-text'
```

Comment:
For some reason the active class isn't applied here

Reply:
I have checked and it works for me, are you talking about the active class for `for_schools` item?